src/main/java/fr/moribus/imageonmap/image/ImageRendererExecutor.java

```diff
@@ -61,46 +61,57 @@
 @WorkerAttributes(name = "Image Renderer", queriesMainThread = true)
 public class ImageRendererExecutor extends Worker
 {
+    static private URLConnection HTTPconnection(final URL url) throws IOException {
```

Comment:
Une méthode qui commence par une majuscule ?
```suggestion
    static private URLConnection httpConnection(final URL url) throws IOException {
```

Reply:
Ça commence par un nom de protocole je ne sais pas comment tu gère ce genre de cas particulier